assets/js/modules/adsense/util/validation.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Validation utilities.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Checks if the given account ID appears to be a valid AdSense account.
+ *
+ * @since n.e.x.t
+ *
+ * @param {string|number} accountID Account ID to test.
+ * @return {boolean} True if the given account ID is valid, false otherwise.
+ */
+export function isValidAccountID( accountID ) {
+	return typeof accountID === 'string' && accountID.match( /^pub-\d+$/ );
```

Comment:
This could use `parseAccountID` and check for a match instead of duplicating the regex. 🤷‍♀

Reply:
It's a different regex, `parseAccountID` is for parsing the account ID from a client ID.